docs/servo_power_test/index.css

```diff
@@ -127,6 +131,10 @@ body {
   position: relative;
 }
 
+#shellScriptList {
```

Comment:
I think we should use kebab-case here for the ids.
https://google.github.io/styleguide/htmlcssguide.html#id_Attributes

Reply:
there are too many lowerCamelCase id, so I will fix only elements which are added in this PR & fix other elements in another PR.

Reply:
(we also have to fix the ids in servo_power_monitor directory...)

Reply:
sounds good, yeah that would be a good choice. thank you!